src/js/components/ServicesTable.js

```diff
@@ -252,6 +273,41 @@ var ServicesTable = React.createClass({
     );
   },
 
+  getRestartConfirmDialog: function () {
+    let message = null;
+    const {state} = this;
+    let service = state.serviceToChange;
+    let buttonText = 'Restart Service';
+
+    if (state.errorMsg) {
+      buttonText = 'Forcefully Restart Service';
+    }
+
+    if (service) {
+      message = (
+        <div className="container-pod flush-top container-pod-short-bottom">
+          <h2 className="text-danger text-align-center flush-top">Restart Service</h2>
+          <p>
+            Are you sure you want to restart <span className="emphasize">{service.getId()}</span>?
+          </p>
+          {this.getErrorMessage()}
+        </div>
+      );
+    }
+
+    return (
+      <Confirm children={message}
```

Comment:
`Confirm` always need children, they cannot be `undefined`. See here: https://github.com/dcos/dcos-ui/pull/808

Also, same thing here with children as props